java/client/test/org/openqa/selenium/remote/http/BUILD.bazel

```diff
@@ -1,10 +1,15 @@
 load("@rules_jvm_external//:defs.bzl", "artifact")
 load("//java:defs.bzl", "java_test_suite")
 
+MEDIUM_TESTS = [
+    "RetryRequestTest.java",
+]
+
 java_test_suite(
     name = "SmallTests",
     size = "small",
-    srcs = glob(["*.java"]),
+    srcs = glob(["*.java"],
+                exclude = MEDIUM_TESTS),
```

Comment:
Unless `buildifier` reformatted things this way, it's fine to put the `exclude` on the same line